### PR TITLE
#14 fix readme examples and better docs

### PR DESCRIPTION
**Pull request by timurista**
Fix the readme. Created new PR as last PR didn't pick up latest changes

Fixes issue: #14

### CODE REVIEW

**Comment by timurista** [comment]
image
Update readme screenshot

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,6 @@ _testmain.go
 *.exe
 *.test
 *.prof
+
+# for testing examples and UI
+default.etcd
```
Comment (ldemailly) [inline .gitignore]: set your editor to add newline to all text files?

File: examples/server_kube/README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
 # Kubernetes ConfigMap example
 
-TODO(michal): Document this.
\ No newline at end of file
+- First download [etcd](https://github.com/coreos/etcd/releases), extract and make it available on your `$PATH`. (might need to move to `/user/local/bin`)
```
Comment (ldemailly) [inline examples/server_kube/README.md]: the server_kube example directory is actually about (file/volume based) configmap updates, etcd isn't needed
(kubernetes is)

File: examples/server/README.md [hunk 3]
```diff
@@ -17,16 +17,16 @@ Set up a set of flags:
...
 ```
 
 Play around by launching the server and visitng [http://localhost:8080](http://localhost:8080):
 
 ```sh
 ./simplesrv &
-etcdctl set /example/flagz/example_my_dynamic_string "I'm santient"
-etcdctl set /example/flagz/example_my_dynamic_int 12345
+etcdctl put /example/flagz/example_my_dynamic_string "I'm santient"
+etcdctl put /example/flagz/example_my_dynamic_int 12345
```
Comment (ldemailly) [inline examples/server/README.md:29]: that's good; does it work though?

File: examples/server_kube/README.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
...
-TODO(michal): Document this.
\ No newline at end of file
+- You need docker and kubernetes. [Docker Desktop](https://www.docker.com/products/docker-desktop), or you can use docker and [minikube](https://minikube.sigs.k8s.io/docs/start/)
+- initialize empty tmp mapping `mkdir -p /tmp/foobar` for kubernetes
+- run the server `go run .`
+
+you should see the following:
+```
+server2020/07/15 13:14:08 Now watching /tmp and /tmp/foobar
+server2020/07/15 13:14:08 etcd flag value watching initialized
```
Comment (ldemailly) [inline examples/server_kube/README.md:10]: your example/doc is correct but the code is wrong
```
	logger.Printf("etcd flag value watching initialized")
```
should be "configmap ..." but let's not change that yet

**Review by ldemailly — APPROVED** [review]

File: examples/server/README.md (hunk 3 above)
Comment (timurista) [reply examples/server/README.md:29]: not yet, it needs to be fixed. Looking at the etcd part

**Comment by ldemailly** [comment]
I didn't notice in the diff the new  examples/server_kube/flagz-debug-view.png, probably should just have reused screenshots/screenshot_endpoint.png to avoid adding more files; will prune that in new version

**Pull request merged**